addons/loki/definitions/log-collect-stdout.cue

```diff
@@ -0,0 +1,1474 @@
+// We put Definitions in definitions directory.
+// References:
+// - https://kubevela.net/docs/platform-engineers/cue/definition-edit
+// - https://kubevela.net/docs/platform-engineers/addon/intro#definitions-directoryoptional
+
+"log-collect-stdout": {
```

Comment:
This definition includes the dashboard for nginx/apache right?

Reply:
only nginx's dashboard